poetry.lock

```diff
@@ -509,17 +509,14 @@ testing = ["pytest (>=6)", "pytest-checkdocs (>=2.4)", "pytest-flake8", "pytest-
 [metadata]
 lock-version = "1.1"
 python-versions = ">=3.7,<4"
-content-hash = "922cdb5dfcf5aa75e5b4f8f85db48545b7ff4ba622d0c2567dcd6d3828c9da19"
+content-hash = "6c00c9c4c6904255d7e204f17f6912f1506866e35334db205ede3fa83a98136a"
 
 [metadata.files]
 anyio = [
     {file = "anyio-3.6.1-py3-none-any.whl", hash = "sha256:cb29b9c70620506a9a8f87a309591713446953302d7d995344d0d7c6c0c9a7be"},
     {file = "anyio-3.6.1.tar.gz", hash = "sha256:413adf95f93886e442aea925f3ee43baa5a765a64a0f52c6081894f9992fdd0b"},
 ]
-attrs = [
-    {file = "attrs-21.4.0-py2.py3-none-any.whl", hash = "sha256:2d27e3784d7a565d36ab851fe94887c5eccd6a463168875832a1be79c82828b4"},
-    {file = "attrs-21.4.0.tar.gz", hash = "sha256:626ba8234211db98e869df76230a137c4c40a12d72445c45d5f5b716f076e2fd"},
-]
+attrs = []
```

Comment:
[high] Update Poetry to the lates version, version < 1.1.14 have a bug which leads to empty list of pinned hashed files

Reply:
Updated - Had to clear the cache: `poetry cache clear --all pypi`

Reply:
Remember to update the `pyproject.toml` and the CHANGELOG